deploy/fleetcommand/README.md

```diff
@@ -87,7 +87,7 @@ echo -n 'AWS_SESSION_TOKEN' | base64
 
 Deploy the Triton Inference Server to your Location in Fleet Command by creating
 a Deployment.  You can specify configuration parameters to override the default
-[values.yaml](values.yaml) in the Application Configuration section.  
+[values.yaml](values.yaml) in the Application Configuration section.
```

Comment:
Need to update the copyright of this file.

Reply:
Thanks, updated these!